README.md

```diff
@@ -28,6 +28,14 @@ Floe can be run as a command-line utility or as a ruby class.
 bundle exec ruby exe/floe --workflow examples/workflow.asl --inputs='{"foo": 1}'
 ```
 
+By default Floe will use `docker` to run `docker://` type resources, but `podman` and `kubernetes` are also supported runners.
+A different runner can be specified with the `--runner` option:
```

Comment:
To match the internals - wondering if this should be --docker-runner and --docker-runner-options.  This way if we need runners for other types in the future there aren't conflicts.